frontend/src/api/metrics/getErrorPercentage.ts

```diff
@@ -0,0 +1,28 @@
+import axios from 'api';
+import { ErrorResponseHandler } from 'api/ErrorResponseHandler';
```

Comment:
can we use single api as we are just passing query in each API

Reply:
Cool will do it!
How about we rename it to file to `getQueryEndpoint.ts` ?